docs/reference/commandline/daemon.md

```diff
@@ -805,6 +805,14 @@ Assuming the daemon is running in cgroup `daemoncgroup`,
 `/sys/fs/cgroup/memory/foobar`, whereas using `--cgroup-parent=foobar`
 creates the cgroup in `/sys/fs/cgroup/memory/daemoncgroup/foobar`
 
+For systemd cgroup, `--cgroup-parent` has different rules. Since systemd
+represents hierarchy by slice and the name of the slice encodes the
+location in the tree. So `--cgroup-parent` for systemd cgroup should be
+a slice name, the name could consists of a dash-separated series of names,
+which describes the path to the slice from the root slice. For example,
+`--cgroup-parent=user-a-b.slice` means the container will be created in
+`<cgroup mount point>/user.slice/user-a.slice/user-a-b.slice/docker-<id>.scope`.
```

Comment:
I think `<cgroup mount point>` would be better, ping @thaJeztah @moxiegirl which one would you prefer?